src/conn/conn_handle.c

```diff
@@ -131,6 +132,9 @@ __wt_connection_destroy(WT_CONNECTION_IMPL *conn)
     __wt_cond_destroy(session, &conn->lsm_manager.work_cond);
 
     /* Free allocated memory. */
+    for (i = 0; i < WT_BLKINCR_MAX; ++i)
+        __wt_free(session, conn->incr_backups[i].id);
```

Comment:
Don't these have to be persistent? That is, if we crash, do callers have to start with a full backup?

Reply:
Yes. That is a later/different ticket to make things persistent. I added a comment/reminder here to say that.